presidio-image-redactor/tests/test_image_analyzer_engine.py

```diff
@@ -385,4 +385,14 @@ def compare_color(actual_pixels, expected_color, threshold=10):
     else:
         assert blue_pixels > 0
 
+def test_check_analyze_supports_language_param(get_mock_png):
+    """Test that the method analyze from class ImageAnalyzerEngine
+    supports the language parameter. 
+    Before there where a bug: 
+        "TypeError: presidio_analyzer.analyzer_engine.AnalyzerEngine.analyze() got multiple values for keyword argument 'language'"
+        
+    :param get_mock_png: The mock PNG image
+    """
+    redacted = ImageAnalyzerEngine().analyze(get_mock_png, language='en')
```

Comment:
Can you please add another test where the language isn't 'en' and the results are empty?

Reply:
Ok. I will do

Reply:
Done.